website/docs/user_guide/personal-api-token.mdx

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
Can you move and rename this file to: `website/docs/how-to/how-to-create-personal-api-tokens.mdx`, please?

It's a how-to guide, so that's where it should go 🤓

Reply:
Definitely willing and able. Honestly this feels like good practise at getting more familiar with the docs which is something I want to do anyway

...but given how much it requires I'm also feeling separate PR